compiler/rustc_errors/src/tests.rs

```diff
@@ -27,10 +27,12 @@ fn make_dummy(ftl: &'static str) -> Dummy {
     let langid_en = langid!("en-US");
 
     #[cfg(parallel_compiler)]
-    let mut bundle = FluentBundle::new_concurrent(vec![langid_en]);
+    let mut bundle: FluentBundle =
+        IntoDyn(crate::fluent_bundle::bundle::FluentBundle::new_concurrent(vec![langid_en]));
```

Comment:
Is it necessary to add the `crate::fluent_bundle::bundle::` qualifier here?

Reply:
`FluentBundle` in the crate doesn't impl `new_concurrent` func so we need qualifier here